bodhi/server/templates/update.html

```diff
@@ -397,6 +397,42 @@
       });
     });
 
+    // handle for waiving test restuls.
```

Comment:
Should this code also only be included if tests are turned on as done below?

Reply:
I don't personally mind the code being here when tests are turned off, since it won't perform any actions without the form being present.

Reply:
Doesn't it make the page heavier ?

Reply:
Yeah it does. I'm certainly not opposed to wrapping it in an if statement.

Reply:
What do you think @matt8754?